internal/rego/load.go

```diff
@@ -0,0 +1,116 @@
+package rego
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/open-policy-agent/opa/ast"
+)
+
+// LoadPoliciesWithAction loads all policies from rego with rules with a given action name
+func LoadPoliciesWithAction(filesContents map[string]string, action string) ([]File, error) {
+	regoFiles, err := loadRegoFiles(filesContents)
+	if err != nil {
+		return nil, fmt.Errorf("load rego files: %w", err)
+	}
+
+	policies := getPoliciesWithAction(regoFiles, action)
+	return policies, nil
+}
+
+// LoadPolicies loads all policies from rego with rules
+func LoadPolicies(filesContents map[string]string) ([]File, error) {
+	regoFiles, err := loadRegoFiles(filesContents)
+	if err != nil {
+		return nil, fmt.Errorf("load rego files: %w", err)
+	}
+
+	policies := getPoliciesWithAction(regoFiles, "")
```

Comment:
This is a pretty good example from the previous discussion about flag arguments. It should be possible to have a `getPolicies()` function, and then `getPoliciesWithAction()` calls that function.

So when we want _all_ policies we call getPolicies()

Reply:
Bad habit :sweat_smile:  Needs to be broken.